### PR TITLE
Switch to internal haskell package repository

### PR DESCRIPTION
**Pull request by disassembler**
- Remove all source-repository-package stanzas (package repository has
  been pre-populated with the exact same versions)
- Remove cabal work-around disabling tests for packages erroneously
  considered local. Cabal now does the right thing
- Remove cabalWrapped, as now unnecessary

### CODE REVIEW

**Review by Jimbo4350 — APPROVED** [review]

File: cabal.project [hunk 1]
```diff
@@ -54,305 +68,82 @@ package cryptonite
...
-package io-classes
-  tests: False
+package comonad
+  flags: -test-doctests
 
-package cardano-ledger-alonzo-test
-  tests: False
+allow-newer:
+  *:aeson,
+  monoidal-containers:aeson,
```
Comment (angerman) [inline cabal.project:114]: ??? if we allow newer aeson anyway, the second line is redundant?

File: cabal.project [hunk 2]
```diff
@@ -54,305 +68,82 @@ package cryptonite
...
--- when using the "cabal" wrapper script provided by nix-shell.
--- --------------------------- 8< --------------------------
--- Please do not put any `source-repository-package` clause above this line.
+-- TODO (mpj): remove this once https://github.com/input-output-hk/ekg-forward/pull/10
+-- is merged and released to CHaP
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/ekg-forward
+  tag: 297cd9db5074339a2fb2e5ae7d0780debb670c63
+  --sha256: 1zcwry3y5rmd9lgxy89wsb3k4kpffqji35dc7ghzbz603y1gy24g
```
Comment (angerman) [inline cabal.project:128]: This has been resolved already as per https://github.com/input-output-hk/cardano-haskell-packages/pull/39?

File: cabal.project [hunk 3]
```diff
@@ -54,305 +68,82 @@ package cryptonite
...
-  type: git
-  location: https://github.com/input-output-hk/ekg-forward
-  tag: 297cd9db5074339a2fb2e5ae7d0780debb670c63
-  --sha256: 1zcwry3y5rmd9lgxy89wsb3k4kpffqji35dc7ghzbz603y1gy24g
-
 source-repository-package
   type: git
   location: https://github.com/denisshevchenko/threepenny-gui
   tag: 4ec92ded05ccf59ba4a874be4b404ac1b6d666b6
   --sha256: 00fvvaf4ir4hskq4a6gggbh2wmdvy8j8kn6s4m1p1vlh8m8mq514
```
Comment (angerman) [inline cabal.project:148]: ??? Why is this in here, there's no comment justifying it.

**Review by angerman — CHANGES_REQUESTED** [review]

File: cabal.project (hunk 2 above)
Comment (andreabedini) [reply cabal.project:128]: That hash is indeed ekg-forward-0.1.0 in CHaP

Comment (andreabedini) [reply cabal.project]: Fixed in [dff694c](https://github.com/input-output-hk/cardano-node/pull/4540/commits/dff694cae80ec2dd8cbec66b19fc82f92236a574)

File: cabal.project (hunk 1 above)
Comment (andreabedini) [reply cabal.project]: Fixed in [dff694c](https://github.com/input-output-hk/cardano-node/pull/4540/commits/dff694cae80ec2dd8cbec66b19fc82f92236a574)

File: cabal.project (hunk 3 above)
Comment (michaelpj) [reply cabal.project:142]: I have nagged Denis and Heinrich about it and hopefully it will go upstream before too long.

Comment (disassembler) [reply cabal.project:142]: It was cherry picked from master.

**Comment by michaelpj** [comment]
Looks like this is going green, any outstanding problems?

**Comment by michaelpj** [comment]
Ugh, it's the hydra corruption error again.

**Pull request merged**